src/client/app/left/date-picker/date-picker.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, OnChanges, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  moduleId: module.id,
+  selector: 'sd-date-picker',
+  templateUrl: 'date-picker.component.html',
+  styleUrls: ['date-picker.component.css']
+})
+
+export class DatePickerComponent implements OnInit, OnChanges {
+  @Output() public datePicked = new EventEmitter();
+  @Input() maxInputDate: string;
+  public max: string = '2017-03-04';
```

Comment:
don't think we can do that. the @Input variable is for our own customised component, here is the data-picker, and max is to pass that to the 'max' property of the input tag. need both variables since there are three layers: home --> data-picker --> input